tests/README.md

```diff
@@ -0,0 +1,128 @@
+# Testing experimental services
+
+The services provided services are covered by unittests. We use GoogleTest as the testing framework and CTest
+as the runner.
+
+When adding a new service please include a unittest suite for your new service in your service PR.
+You are free to use any framework as long as it integrates with our CTest runner but to take advantage of
+mocks provided by mbed-os you'll have to use [GoogleTest](https://github.com/google/googletest).
+
+# Running
+
+To run unittests simply run:
+
+```
+./unittests.sh
+```
+
+This will configure and build the tests with Cmake and run the tests.
+
+Unit tests require mbed-os to provide stubs and mocks. A copy of mbed-os will be checked out during the build into
+the tests directory (unless it's already there, you may use a symlink if you already have a checkout of mbed-os).
+This will not be deleted after the run.
```

Comment:
This _might_ be a source of confusion for people. Perhaps you could add the folder structure:
```
tests/
└── CI/
└── UNITESTS/
└── mbed-os/
.mbedignore
CMakeLists.txt
googletest.in
unittests.sh
```

Reply:
OK, I expanded that paragraph.

Reply:
Resolved in 54b5a6f18b17b62799b6973382344e625814ded3